README.md

```diff
@@ -1,2 +1,180 @@
-# jbang-catalog
+# Jython JBang Catalog
+
 Catalog for the JBang command line launcher (https://jbang.dev)
+
+## Usage
+
+To use the JBang scripts from this catalog you need to have [JBang](https://www.jbang.dev/) installed.
+
+To start the Jython interpreter or run a Jython script via the interpreter, use JBang as follows and add additional options as required:
+
+* `jbang run jython-cli@jython <jython-script>.py`
+
+## App
+
+The `jython-cli@jython` JBang script can also be installed as a JBang application which is accessible from the command-line.
+
+```
+jbang app install jython-cli@jython
+[jbang] Command installed: jython-cli
+```
+
+Jython scripts can be run using the `jython-cli` application as:
+
+* `jython-cli <jython-script>.py`
+
+On supported platforms a Jython script (with execution permissions enabled) can also be started as
+
+`./<jython-script>.py`
+
+provided that the first line of the script contains text:
+
+`#!/usr/bin/env jython-cli`
+
+## PEP-723
+
+`jython-cli` uses the `PEP-723` inline script metadata format to specify:
+
+* The version of Jython to use: 2.7.1, ... 2.7.4
+* The major version of Java to use: 17, 21, ...
+* Maven dependencies to make available to the Jython script
+* Additional Java runtime options to use
+
+[Jython metadata for JBang]
+```
+# /// jbang
+# requires-jython = ">=2.7.4"
+# requires-java = ">=21"
+# dependencies = [
+#   "io.leego:banana:2.1.0"
+# ]
+# [java]
+#   runtime-options = "-server -Xms2g -Xmx2g -XX:+UseZGC -XX:+ZGenerational"
+# ///
+```
+
+[Jython script banner.py with metadata for JBang]
+
+```python
+#!/usr/bin/env jython-cli
+
+# banner.py
+
+# /// jbang
+# requires-jython = ">=2.7.4"
+# requires-java = ">=21"
+# dependencies = [
+#   "io.leego:banana:2.1.0",
+# ]
+# [java]
+#   runtime-options = ""
+# ///
+
+import sys
+
+import io.leego.banana.BananaUtils as BananaUtils
+import io.leego.banana.Font as Font
+
+def main():
+    print(sys.argv)
+
+    text0 = "Jython 2.7"
+    text1 = BananaUtils.bananaify(text0, Font.STANDARD)
+
+    print(text1)
+
+main()
+```
+
+## Example 1 - run Jython and display its version number
+
+```bash
+jbang run jython-cli@jython --version
+```
+
+```
+2.7.4.0
```

Comment:
Okay, I see. This script is Jython 2.7.4-specific and has a nano-version component to track the script version. Does this mean when Jython 2.7.5 is released and let's say this script got multiple iterations, whould after 2.7.4.23 come 2.7.5.0 or 2.7.5.24? Perhaps this version semantics should be explained for future maintainers.

Reply:
> 2.7.4.23 come 2.7.5.0

When the Jython version number changes, the FIX_NUMBER is reset to 0. Will update the documentation accordingly.

Reply:
Fixed.